### PR TITLE
Fixed bug when trying to remove pkgs in test mode with progress

### PR DESCRIPTION
**Pull request by zraul123**
### What does this PR do?
Fixes a bug when trying to run simulated (test) jobs that remove packages

### CODE REVIEW

File: salt/modules/opkg.py [hunk 1]
```diff
@@ -833,8 +833,8 @@ def _execute_remove_command(cmd, is_testmode, errors, jid):
             errors = [out['stderr']]
         else:
             errors = [out['stdout']]
-    else:
-        errors = []
+    elif is_testmode:
+        reportedPkgs = _parse_reported_packages_from_remove_output(out['stdout'])
```
Comment (chotea) [inline salt/modules/opkg.py:838]: Why did you delete "errors = []" ?. Maybe there are callers that send None for this so they expect an array in return.

**Review by chotea — APPROVED** [review]

**Review by andzn — APPROVED** [review]
